src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -469,7 +466,7 @@ public void DiagnosticSourceStartStop()
                     Assert.InRange(observer.Activity.Duration, TimeSpan.FromTicks(1), TimeSpan.MaxValue);
 
                     // let's only check that Duration is set in StopActivity, we do not intend to check precision here
-                    Assert.InRange(observer.Activity.StartTimeUtc + observer.Activity.Duration, DateTime.MinValue, DateTime.UtcNow.AddMilliseconds(2 * MaxClockErrorMSec));
+                    Assert.InRange(observer.Activity.Duration, TimeSpan.MinValue, stopWatch.Elapsed.Add(TimeSpan.FromMilliseconds(2 * MaxClockErrorMSec)));
```

Comment:
Probably not actually.  I forget that it's negative.  I've switched it to TimeSpan.Zero + 1 tick as the minimum duration.